md-dao/src/main/java/com/wipro/ats/bdre/md/dao/AppPermissionDAO.java

```diff
@@ -0,0 +1,42 @@
+package com.wipro.ats.bdre.md.dao;
+
+import com.wipro.ats.bdre.md.dao.jpa.PermissionType;
+import org.apache.log4j.Logger;
+import org.hibernate.Criteria;
+import org.hibernate.Session;
+import org.hibernate.SessionFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.List;
+
+/**
+ * Created by cloudera on 4/6/16.
+ */
+@Transactional
+@Service
+public class AppPermissionDAO {
+    private static final Logger LOGGER = Logger.getLogger(AppPermissionDAO.class);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "LOGGER" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1068)